examples/qm9_pretrained_schnet.py

```diff
@@ -19,10 +52,10 @@
     loader = DataLoader(test_dataset, batch_size=256)
 
     maes = []
-    for data in loader:
+    for data in tqdm(loader):
         data = data.to(device)
         with torch.no_grad():
-            pred = model(data.z, data.pos, data.batch)
+            pred = predict(model, data)
```

Comment:
```suggestion
            if not args.use_precomputed_edges:
                 pred = model(data.z, data.pos, data.batch)
            else:
                 pred = model(data.z, data.pos, data.batch, data.edge_index)
```
and drop `default_predict` and `precomputed_predict`.